### PR TITLE
#DP-9 ft: Admin dashboard

### PR DESCRIPTION
**Pull request by jeandamascyifuzo**
### PR Description
This Pr shows the progress of the Admin dashboard. 

### Description of the tasks to be completed
- [x] Add and remove programs/cohorts
- [x] Add and remove session for attendance
- [x] Manage roles

### How has this been tested?
clone Rep
git checkout ft-admin-dashboard-dp-9
git fetch
npm install
npm run dev

or

**https://devpulse-review-app-28.herokuapp.com/dashboard/admin/
then go to
cohorts
session
manage role**

### Relevant trello story:
[Trello](https://trello.com/c/KCXhcD9T/9-admin-dashboard)

### CODE REVIEW

**Comment by feyton** [comment]
I deployed a review app for you to try out this pull request 👇🏽

Heroku App: https://devpulse-review-app-28.herokuapp.com/


<hidden purpose="for-rewritable-pr-comment-action-use" value="Deploy"></hidden>

**Review by chanceDira — COMMENTED** [review]
Good work, 

On my end am getting margin bottom issue

Screen Shot 2022-07-18 at 08 36 11

File: src/App.tsx [hunk 1]
```diff
@@ -7,12 +7,14 @@ import './index.css';
...
   return (
     <div className="flex flex-col min-h-screen ">
-      <Router>
+       <Router>
         <Routes>
           <Route path="/dashboard/*" element={<DashRoutes />} />
           <Route path="/*" element={<MainRoutes />} />
         </Routes>
       </Router>
+      {/* <Dash/> */}
```
Comment (samyshyaka) [inline src/App.tsx:16]: You can remove the commented code

File: src/containers/admin-dashBoard/Cohorts.tsx [hunk 2]
```diff
@@ -0,0 +1,318 @@
...
+                                                    </p>
+                                                </td>
+                                                <td className="px-5 py-5 border-b border-gray-200 bg-white dark:bg-dark-bg text-sm">
+                                                    <p className="text-gray-900 dark:text-white whitespace-no-wrap">
+                                                        11th Dec 2022
+                                                    </p>
+                                                </td>
+                                                <td className="px-5 py-5 border-b border-gray-200 bg-white dark:bg-dark-bg text-2xl">
+                                                    <p className="text-gray-900 dark:text-white whitespace-no-wrap">
+                                                        ...
```
Comment (samyshyaka) [inline src/containers/admin-dashBoard/Cohorts.tsx:234]: Remember to include the dropdown for actions

**Review by samyshyaka — CHANGES_REQUESTED** [review]
Nice work @jeandamascyifuzo ,

- Try to center your cards
- Remember to put drop-downs where they are missing

**Review by chanceDira — APPROVED** [review]

**Review by samyshyaka — CHANGES_REQUESTED** [review]
Find out why the builds are failing and try to fix them

**Review by Julesbi — APPROVED** [review]

**Review by IIshimwe — APPROVED** [review]

**Review by ceelogre — COMMENTED** [review]
Coverage is very low.

**Review by ceelogre — COMMENTED** [review]
What's the URL to test this?

**Review by ceelogre — CHANGES_REQUESTED** [review]
The elements are not aligned on the left. Also, you don't need to translate certain words like `cohorte` and `concept de base`
Capture d’écran 2022-07-21 à 17 44 24

**Review by samyshyaka — APPROVED** [review]

**Review by ceelogre — APPROVED** [review]

**Pull request merged**